src/graphql/resolvers/businessInfoResolver.ts

```diff
@@ -1,13 +1,24 @@
+import { BusinessImageProviderAggregator } from '../../providers/data_providers/businessImageProviderAggregator';
+import {
+	ExtractionConfig,
+	getExtractionProvider,
+} from '../../providers/extraction_providers/extractionProviderFactory';
+import { getInterpretationProvider, InterpretationConfig } from '../../providers/interpretation_providers';
 import { BusinessInferences, BusinessInfoInput } from '../generatedTypes';
 
 export async function getBusinessInfo(businessInfo: BusinessInfoInput): Promise<BusinessInferences> {
-	return {
-		servesAlcohol: {
-			insight: false,
-			confidence: 0.5,
-			evidence: {
-				images: [],
-			},
-		},
-	};
+	if (!process.env.EXTRACTION_CONFIG) throw new Error('Undefined environment variable EXTRACTION_CONFIG');
+	if (!process.env.INTERPRETATION_CONFIG)
+		throw new Error('Undefined environment variable INTERPRETATION_CONFIG');
```

Comment:
We can have a private function to perform this check and throw errors, might make it more readable when we have more env variables

Reply:
I wonder if there's an environment variable parsing library for this purpose?

Reply:
There is one called [envalid](https://github.com/af/envalid) that seems to do what we want here. There may also be a way to use the [2nd answer of this stackoverflow post](https://stackoverflow.com/questions/45194598/using-process-env-in-typescript).